### PR TITLE
Create secret yaml file

### PR DESCRIPTION
**Pull request by mwisnia97**
**Description**
We want to only apply all YAML files with configuration to configure new cluster.

**Changes**
- Add secret YAML file

### CODE REVIEW

File: resources/secrets.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    - manager: kubectl-create
+      operation: Update
+      apiVersion: v1
+      fieldsType: FieldsV1
+      fieldsV1:
+        f:data:
+          .: {}
+          f:REDIS_PASSWORD: {}
+          f:GCP_API_KEY: {}
+          f:GCP_EMAIL: {}
```
Comment (pPrecel) [inline resources/secrets.yaml:17]: Do we really need these fields? Look like a combination of parameters injected by the Kubernetes and then copied to the file

File: resources/secrets.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+data:
+  GCP_API_KEY: >-
+    {YOUR_GCP_API_KEY}
+  GCP_EMAIL: {YOUR_GCP_EMAIL}
+  REDIS_PASSWORD: {YOUR_REDIS_PASSWORD}
```
Comment (pPrecel) [inline resources/secrets.yaml:22]: Do you have any method/technology to inject true values to the fields like this: {YOUR_GCP_API_KEY}?

File: resources/secrets.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+      apiVersion: v1
+      fieldsType: FieldsV1
+      fieldsV1:
+        f:data:
+          .: {}
+          f:REDIS_PASSWORD: {}
+          f:GCP_API_KEY: {}
+          f:GCP_EMAIL: {}
+data:
+  GCP_API_KEY: >-
```
Comment (pPrecel) [inline resources/secrets.yaml:19]: What does this `>--` mean?

**Review by pPrecel — COMMENTED** [review]
A few questions:

File: resources/secrets.yaml (hunk 2 above)
Comment (pPrecel) [reply resources/secrets.yaml:22]: Additionally, if not and you base on tutorials like [this](https://github.com/kyma-incubator/Kyma-Showcase/blob/main/README.md) (installation - step 2) then please add information about this to the README.md. Changes like this should be implemented together with information on how to use it.

**Review by pPrecel — APPROVED** [review]
/meow LGTM

**Review by mvshao — APPROVED** [review]

**Review by Lyczeq — APPROVED** [review]

**Pull request merged**